modules/dns_spoof/dns_spoof.go

```diff
@@ -80,17 +99,44 @@ func (mod DNSSpoofer) Author() string {
 	return "Simone Margaritelli <evilsocket@gmail.com>"
 }
 
+func (mod *DNSSpoofer)getSystemResolver() (net.IP,error) {
```

Comment:
this won't work on platforms that are not GNU/Linux i think ...

Reply:
It does not work very well on Linux either, because it just uses the first nameserver occurrance in hardcoded and therefore the setting "system" on dns.spoof.proxy.srv is more a convenience feature for me. maybe a default of 8.8.8.8 might be better (instead of "system" as default).

Reply:
why not just another parameter to let the user specify the default resolver?

Reply:
I think thats already possible, if i understand u correctly.
dns.spoof.proxy.srv can either be set to 'system' (linux/mac only) to use the system-dns for proxing or _an ip_.